code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -984,7 +984,6 @@ datum
 								cube_mob(M,volume_passed)
 						else
 							cube_mob(M,volume_passed)
-				M.bodytemperature = 0
 
```

Comment:
Seems pretty harsh to completely remove its ability to cool people down on touch. Even if it's not turning them into ice cubes I feel like it should still cool them somewhat. With this it would still take more than 100u to bring them down to 0K.

```suggestion
M.bodytemperature = max(M.bodytemperature - (volume_passed * 2), 0)

```